rules/linux/credential_access_potential_linux_ssh_bruteforce.toml

```diff
@@ -0,0 +1,81 @@
+[metadata]
+creation_date = "2022/09/14"
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2022/09/14"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies multiple consecutive login failures targeting an user account from the same source address and within a
+short time interval. Adversaries will often brute force login attempts across multiple users with a common or known
+password, in an attempt to gain access to accounts.
+"""
+from = "now-9m"
+index = ["auditbeat-*", "logs-system.auth-*"]
```

Comment:
can u confirm that the rule trigger on `auditbeat-*` ? if not u can remote it for now.

Reply:
This is triggering for auditbeat logs as well and the query has been modified to include user_login auditbeat metadata and the test logs have been updated in the issue.